nearcore/src/runtime/snapshot_grabber.rs

```diff
@@ -0,0 +1,92 @@
+use crate::NightshadeRuntime;
+use borsh::BorshSerialize;
+use near_chain::RuntimeAdapter;
+use near_primitives::hash::CryptoHash;
+use near_primitives::state_record::{state_record_to_account_id, StateRecord};
+use near_primitives::transaction::SignedTransaction;
+use near_primitives::types::{AccountId, EpochId, ShardId, StateRoot};
+use near_store::TrieIterator;
+use std::cell::Cell;
+use std::collections::HashSet;
+use std::fs;
+use std::fs::File;
+use std::sync::Mutex;
+use tracing::debug;
+
+pub struct SnapshotGrabber {
```

Comment:
how do you use this tool?

I was expecting a stand-alone binary, which you can 'point' at the existing database, and pass some parameters (like which accounts to take etc).